From d8c8068cf55a7e528f8e979b02d620a053b53f82 Mon Sep 17 00:00:00 2001
From: "Justin M. Keyes" <justinkz@gmail.com>
Date: Sun, 27 Apr 2025 15:49:00 -0700
Subject: [PATCH] Revert "fix(desktop): cannot open filename with spaces using
 OS file manager" #33684

This reverts commit 6e12ef4a7b4cd966244ac6bce2593943e7df7758

> Paths with spaces were already working. The original bug is most
> likely with user's terminal desktop entry, file manager or DE, and has
> nothing to do with nvim.desktop.

These are 3 different implementations that work correctly with unquoted %F and spaces:
```
$ DE=generic xdg-open "D I R/F I L E.txt" # pure bash
$ gio open "D I R/F I L E.txt" # glib2
$ handlr open "D I R/F I L E.txt" # rust
```

(cherry picked from commit 07a207a5f16be98053a56cc0a20d182717deaf54)
---
 runtime/nvim.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/nvim.desktop b/runtime/nvim.desktop
index 50326fa618e003..f9feae926db76f 100644
--- a/runtime/nvim.desktop
+++ b/runtime/nvim.desktop
@@ -78,7 +78,7 @@ Comment[wa]=Asspougnî des fitchîs tecses
 Comment[zh_CN]=编辑文本文件
 Comment[zh_TW]=編輯文字檔
 TryExec=nvim
-Exec=nvim "%F"
+Exec=nvim %F
 Terminal=true
 Type=Application
 Keywords=Text;editor;
